gradle/libs.versions.toml

```diff
@@ -1,5 +1,5 @@
 [versions]
-agp = "8.9.1"
+agp = "8.5.1"
```

Comment:
For some reason, I had to make this change to get this to work. Dunno why. Probably was something specific to me.
```suggestion
```